src/flinkSql/socketConnectionManager.ts

```diff
@@ -0,0 +1,331 @@
+import { EventEmitter } from "vscode";
+import { Message, MessageTransports } from "vscode-languageclient/node";
```

Comment:
we're not using the custom MessageTransports here because it's at the socket level not the server provision level I imagine?